README.md

```diff
@@ -35,6 +35,14 @@ const declaration = {
 	line: 1, column: 1
 }
 
+const comment = {
+	value: '/*@noflip*/',
+	type: 'comm',
+	props: '/',
```

Comment:
what are possible `props` by design? by running this:
```js
compile(`.user {
  --foo: 10px; // test
}`)
```
I see that inline comment is normalized to a block comment (getting `value: '/* test*/'`) which is fine by me as discussed previously. I'm only wondering if `props: '\n'` is by design for this case.

Reply:
Yes, `/` is for block comments and '\n' for inline comments noted by the last character  used to terminate them.

Reply:
gotcha, cool 👍